.docker/jats-ingester/airflow.cfg

```diff
@@ -23,6 +23,9 @@ temp_directory_path =
 # url of libero search service endpoint used to reindex items
 search_url = http://api-gateway:8081/search/populate
 
+# number of thread pool workers
+thread_pool_workers =
```

Comment:
should it have a number, right? what's the default?

Reply:
If left empty it will use the python default of `(os.cpu_count() or 1) * 5` https://github.com/python/cpython/blob/3.7/Lib/concurrent/futures/thread.py#L127

Reply:
Would you like me to set a default?

Reply:
Let's go for the Python default